### DISCUSSION TITLE
How do I build local-report-base.js ?

### DISCUSSION THREAD

**Discussion by akdor1154** [datapane/datapane#183] [Help] [answered]
Hey guys,
Sorry if I'm missing something obvious.

I want to play around with an idea I have for local interactivity, but I can't find where/how local-report-base.js gets built.

Is the source to this available or is it proprietary? I am seeing hints in your gh history of something being split out in the past?

Cheers
Jarrad

**Comment by mands** [comment, marked as answer]
Hi @akdor1154 ,

Thanks for your question - right now `local-report-base.js` is currently proprietary - we're looking at open-sourcing in the near future, we just haven't had a moment to step back and do all the work involved.

**Reply by akdor1154** [reply]
No worries, thanks for the clarification, and good luck - it seems pretty difficult to have a viable company while still open sourcing all your work :)
